component/prometheus/exporter/blackbox/blackbox_test.go

```diff
@@ -67,6 +72,40 @@ func TestUnmarshalRiverWithInlineConfig(t *testing.T) {
 	require.Contains(t, "http://grafana.com", args.Targets[1].Target)
 	require.Contains(t, "http_2xx", args.Targets[1].Module)
 }
+
+func TestUnmarshalRiverWithInlineConfigYaml(t *testing.T) {
+	riverCfg := `
+		config = "modules:\n  http_2xx:\n    prober: http\n    timeout: 5s\n"
```

Comment:
I decided to include this similar test since it validates and demonstrates the traditional yaml format in use for this argument.